Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsSetupViewportUiCluster.cpp

```diff
@@ -196,7 +225,6 @@ namespace EMotionFX
             AzToolsFramework::ViewportUi::ViewportUiRequestBus::Event(
                 GetViewportId(), &AzToolsFramework::ViewportUi::ViewportUiRequestBus::Events::RemoveCluster, m_colliderClusterId);
             m_colliderClusterId = AzToolsFramework::ViewportUi::InvalidClusterId;
-            m_subModes[m_subMode]->Teardown();
```

Comment:
question: no need to call this anymore?

Reply:
I wanted to make sure it was always paired with setup, so now it only happens in SetCurrentMode. I also added a null mode so you can set that to indicate no other mode is active.